wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -577,9 +577,11 @@ def bulk_action_choices(context, app_label, model_name):
                     )
                     + "?"
                     + urlencode({"next": next_url}),
-                    attrs={"aria-label": action.aria_label},
+                    attrs={
+                        "aria-label": action.aria_label,
+                        "data-bulk-action-button": "",
+                    },
                     priority=action.action_priority,
-                    classname="bulk-action-btn",
```

Comment:
As per the comment in:
https://github.com/wagtail/wagtail/blob/a2b0d04563ce9f4f5c1ca1916a64c724aab0adb4/wagtail/admin/widgets/button.py#L57-L63

and the rendering in:
https://github.com/wagtail/wagtail/blob/a2b0d04563ce9f4f5c1ca1916a64c724aab0adb4/wagtail/admin/templates/wagtailadmin/pages/listing/_dropdown_items.html#L4

this `bulk-action-btn` class is never rendered for the buttons inside the dropdown. To test, you can change the bulk actions limit here: https://github.com/wagtail/wagtail/blob/33055f3fbf17db68c9a32ac7fac357af84dbcc97/wagtail/admin/templatetags/wagtailadmin_tags.py#L544-L545

to e.g. 2, then observe:

<img width="952" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/1de9422c-1d6e-46a9-8f31-28cf3010abdd">


Hence I'm changing the approach to use data attributes instead.